docs/en/supported_models/supported_models.md

```diff
@@ -28,3 +28,4 @@
 | ChatGLM2  |    6B     |    Yes    |   No    |  No  |
 |  Falcon   | 7B - 180B |    Yes    |   No    |  No  |
 |    YI     | 6B - 34B  |    Yes    |   No    |  No  |
+|  Mistral  |    7B     |    Yes    |   No    |  No  |
```

Comment:
Could update the readme as well:
https://github.com/InternLM/lmdeploy/blob/main/README.md#supported-models